src/iris/vendors/iris_slack.py

```diff
@@ -125,8 +126,17 @@ def send_message(self, message):
                     return time.time() - start
                 # If message is invalid:
                 #   {u'ok': False, u'error': u'invalid_arg_name'}
-                logger.error('Received an error from slack api: %s',
-                             data['error'])
+
+                # Check the slack api to see if the iris bot is in the channel first before logging an error
+                # if not in the channel, the error is expected and should not be logged
+                msg_endpoint = self.config['base_url'] + "/conversations.info"
```

Comment:
this api call is unnecessary, slack api already returns a `not_in_channel error`, you should be able to the same thing we do above in line 123